src/amber/cli/templates/scaffold/controller/granite/src/controllers/{{name}}_controller.cr.ecr

```diff
@@ -63,4 +63,14 @@ class <%= @name.capitalize %>Controller < ApplicationController
     end
     redirect_to "/<%= @name %>s"
   end
+
+  def <%= @name %>_params
+	params.validation do
+	  <% @fields.each do |field| %>
+	  <% unless ["created_at", "updated_at"].includes? field.name %>
+	  required(:<%= field.name %>) { |p| !p.as(<%= field.cr_type.capitalize %>) }
```

Comment:
Shouldn't it be `as?` instead of `as` ? `as` rises an exception which would result in 500 error code instead of validation error.

Reply:
I personally prefer the exception knowing that is not being cast to the type I am expecting, but it can be changed to `as?`, I guess  it depends in the use case. IMHO.

Reply:
@eliasjpr imagine a view with a form that requires integer value. User enters "blah" and submits. In my understanding controller will throw exception and user will be responded with status code 500, instead of a flash message "you have to enter integer value". Am I wrong?